### PR TITLE
ff: fix menu on proposal details

### PR DESCRIPTION
**Pull request by MarkNerdi**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

https://github.com/iotaledger/firefly/pull/7121
https://github.com/iotaledger/firefly/pull/7158

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/components/ProposalsDetails.svelte [hunk 1]
```diff
@@ -62,11 +62,11 @@
 </script>
 
 <proposals-details class="space-y-4">
-    <header-container class="flex justify-between items-center">
+    <header-container class="flex justify-between items-center relative">
         <Text fontSize="14" fontWeight={FontWeight.semibold}>
             {localize('views.governance.proposalsDetails.title')}
         </Text>
-        <ProposalsDetailsButton />
+        <ProposalsDetailsButton modalPosition={{ right: '0px', top: '34px' }} />
```
Comment (Tuditi) [inline packages/desktop/components/ProposalsDetails.svelte:69]: Missing lines of code here:
image

File: packages/shared/src/components/modals/ProposalsDetailsMenu.svelte [hunk 2]
```diff
@@ -1,26 +0,0 @@
...
-<Modal bind:this={modal} {...$$restProps}>
-    <div class="flex flex-col">
-        <MenuItem
-            icon={Icon.Delete}
-            iconProps={{ width: '16', height: '19' }}
-            title={localize('actions.revote')}
-            onClick={onRevoteClick}
-        />
-    </div>
-</Modal>
```
Comment (Tuditi) [inline packages/shared/src/components/modals/ProposalsDetailsMenu.svelte:26]: Why did we remove this instead of ProposalsDetailsButton?

**Review by Tuditi — CHANGES_REQUESTED** [review]
I have the feeling that we changed things here with regard to how FF did it. This is not necessarily a problem, but the original issue isn't fixed. To test register to all events on https://firefly.h.testnet.shimmer.network.

image

Maybe in this case it's better to follow the same approach as FF did?

File: packages/desktop/components/ProposalsDetails.svelte (hunk 1 above)
Comment (MarkNerdi) [reply packages/desktop/components/ProposalsDetails.svelte:69]: this outer div isnt required, so even though they added it, i removed it

File: packages/shared/src/components/modals/ProposalsDetailsMenu.svelte (hunk 2 above)
Comment (MarkNerdi) [reply packages/shared/src/components/modals/ProposalsDetailsMenu.svelte:26]: so we have the similar structure as with the other `ProposalDetailsButton`

<MeatBallButton />
<Modal>
   <div/>
</Modal>

**Review by Tuditi — DISMISSED** [review]
LGTM! Deduplicate the ModalPosition type, since it resulted in a minor type issue. Have a look and merge if you like it. :relaxed:

**Review by Tuditi — APPROVED** [review]

**Pull request merged**